Makefile

```diff
@@ -32,11 +32,15 @@ image_base="ubuntu"
 image_tag="20.04"
 endif
 
+buildx=0
+cache_from=type=local,src=/tmp/.buildx-cache
+cache_to=type=local,dest=/tmp/.buildx-cache
 ### function for building
 ### $(1) is name
 ### $(2) is dockerfile filename
 ### $(3) is package type
 ### $(4) is code path
+ifneq ($(buildx), "true")
```

Comment:
@qsliu2017 Hi, thanks for the update. While I would prefer to use `True`/`False`, instead of `true`/`false` to keep align with other params, as you mentioned before. Thanks.

Reply:
@qsliu2017 Hi, looking forward to your feedback. Thanks. 😄